packages/ember/package.json

```diff
@@ -21,7 +21,7 @@
     "clean": "yarn rimraf sentry-ember-*.tgz dist tmp build .node_modules.ember-try package.json.ember-try",
     "lint": "run-p lint:js lint:hbs lint:ts",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint . --cache --cache-location '../../eslintcache/'",
```

Comment:
Is this intentional?

Reply:
Yes, we removed all of these in a previous PR, just forgot about this. (Stuff I find when trying to debug stuff locally)